lib/oxidized/core.rb

```diff
@@ -31,6 +31,7 @@ def initialize args
     private
 
     def run
+      Oxidized.logger.debug "lib/oxidized/core.rb: Starting the worker..."
```

Comment:
path of the file, but yeah, I think it's useful  to people who want to fix something but are not intimately aware with the codebase.